opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -34,8 +34,9 @@
     /**
      * Sets the end timestamp to now and records the span.
      *
-     * <p>With the exception of calls to Span.context(), this should be the last call made to the span instance, and to
-     * do otherwise leads to undefined behavior.
+     * <p>With the exception of calls to {@link #context}, this should be the last call made to the span instance.
+     * Future calls to {@link #finish} are defined as noops, and future calls to methods other than {@link #context}
```

Comment:
I added this comment text to define multiple `finish()` calls as noops... that way users can `finish()` manually and have the `ActiveSpan` try-with-resources scope end afterwards (without causing undefined behavior).